requirements.txt

```diff
@@ -1,28 +1,20 @@
 bech32==1.2.0
 bitstring==3.1.6
+brotli==1.0.7
 cerberus==1.3.2
 certifi==2020.4.5.1
 chardet==3.0.4
-click==7.1.1
+click==7.1.2
 flask-assets==2.0
-flask-compress==1.4.0
+flask-compress==1.5.0
 flask-talisman==0.7.0
 flask==1.1.2
-gevent==20.4.0
-googleapis-common-protos==1.51.0
-greenlet==0.4.15 ; platform_python_implementation == 'CPython'
-grpcio-tools==1.28.1
-grpcio==1.28.1
-gunicorn==20.0.4
```

Comment:
Were these ever needed as direct deps? I don't see them removed in the code/encapsulated with `try` anywhere.

Reply:
`lnd-grpc` requires `grpcio`, `grpcio-tools`, `googleapis-common-protos`.

I removed the wgsi choice from the core library too: when deploying users should install `gunicorn` and `gevent` (`greenlet` is rewuired/installed by the latter). But I am not sure if we should keep them or remove them. You can run LNbits with `flask run` but in production is recommended to use a wgsi server line gunicorn.

In the end, it seems that the RaspiBlitz willl finally be using the gRPC libraries, so this PR can be ignored for now, and we can leave all dependencies there, even if some of them might never be used by the user...

Reply:
Regarding the extra CI: I am not used to GitHub Actions, I didn't realize I was adding another CI :) if we want to only work with GiHub Actions, is OK for me too, just moving the tests there.